### PR TITLE
Add files prop to allow specifying which files to include assets in.

### PR DESCRIPTION
**Pull request by jaridmargolin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ function isArray (v) {
   return Array.isArray(v);
 }
 
+function castArray (v) {
+  return isArray(v) ? v : [v];
+}
+
```
Comment (jharris4) [inline index.js]: Can you follow the pattern used by option.assets, where the files gets converted to an array only if it is a String? That way things will be more consistent

File: index.js [hunk 2]
```diff
@@ -77,6 +82,10 @@ HtmlWebpackIncludeAssetsPlugin.prototype.apply = function (compiler) {
   // Hook into the html-webpack-plugin processing
   compiler.plugin('compilation', function (compilation) {
     compilation.plugin('html-webpack-plugin-before-html-generation', function (htmlPluginData, callback) {
+      if (self.options.files && !castArray(self.options.files).includes(htmlPluginData.outputName)) {
+        return callback(null, htmlPluginData);
+      }
+
```
Comment (jharris4) [inline index.js]: Array.includes does not seem to be supported in older versions of Node (that seems to be why the Travis tests were failing). Could you switch this to `files.indexOf(htmlPluginData.outputName) !== -1` instead ?

**Comment by jaridmargolin** [comment]
Thanks for the quick review... I'll get the suggested updates pushed up shortly.

**Comment by jaridmargolin** [comment]
I addressed a few of your concerns. There does appear to be failing tests on 0.12 however I don't believe this version of node is even [supported by webpack](https://github.com/webpack/webpack/blob/master/.travis.yml#L25) any more.

If you are 100% insistent on supporting, I can spend a little more time looking into what is happening, however it is not immediately clear what is causing the failure.

**Comment by jharris4** [comment]
Thanks  for addressing my concerns! :-)

The example you added actually helped me to understand the motivation for why the files option is useful, so that's great...

I guess I'm not concerned about node 0.12. Technically this plugin was supporting webpack 1.x and node 0.12 with your change, but I'm fine with dropping support for node 0.12. I've updated the travis config accordingly.

**Comment by jharris4** [comment]
Published in Version 0.0.5.

**Pull request merged**